scripts/generate-google3-docs/resource-reference/templates/cloudbuild_cloudbuildworkerpool.tmpl

```diff
@@ -0,0 +1,50 @@
+{{template "headercomment.tmpl" .}}
+
+{% extends "config-connector/_base.html" %}
+
+{% block page_title %}{{ .Kind}}{% endblock %}
+{% block body %}
+{{template "alphadisclaimer.tmpl" .}}
+<table>
+<thead>
+<tr>
+<th><strong>Property</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Name</td>
+<td>Cloud Build</td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Documentation</td>
+<td><a href="/cloud-build/docs/">/cloud-build/docs/</a></td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} REST Resource Name</td>
+<td>v1.projects.workerpools</td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} REST Resource Documentation</td>
+<td><a href="/cloud-build/docs/api/reference/rest/v1/projects.locations.workerPools">/cloud-build/docs/reference/rest/v1/projects.locations.workerPools</a></td>
+</tr>
+<tr>
+<td>{{"{{product_name_short}}"}} Resource Short Names</td>
+<td>{{ .ShortNames}}</td>
+</tr>
+<tr>
+<td>{{"{{product_name_short}}"}} Service Name</td>
+<td>cloudbuild.googleapis.com</td>
+</tr>
+<tr>
+<td>{{"{{product_name_short}}"}} Resource Fully Qualified Name</td>
+<td>{{ .FullyQualifiedName}}</td>
+</tr>
+{{template "iamsupport.tmpl" .}}
+</tbody>
```

Comment:
Looks like you missed the following section after line 44:

```
<tr>
<td>{{"{{product_name_short}}"}} Default Average Reconcile Interval In Seconds</td>
<td>{{ .DefaultReconcileInterval}}</td>
</tr>
```
Is that intentional?

Reply:
Thanks. Yeah, I don't think cbwp needs to emphasize that (too much info sometimes dilute the most valuable pieces). But I'm okay to put it back for consistency.